test/common_utils.py

```diff
@@ -102,11 +102,11 @@ def cycle_over(objs):
 
 
 def int_dtypes():
-    return torch.testing.integral_types()
+    return (torch.uint8, torch.int8, torch.int16, torch.int32, torch.int64)
 
 
 def float_dtypes():
-    return torch.testing.floating_types()
+    return (torch.float32, torch.float64)
```

Comment:
Should we include `torch.float16` and `torch.bfloat16` here?

Reply:
Sometimes I had to check float16 (see [example](https://github.com/pytorch/vision/blob/90a2402b11b58705884b564b7bfce325f1c7d7c7/test/test_ops.py#L953-L958)) in the test to ensure the operator implementation did not overflow. We don't use the specific method there but maybe it's going to be useful?

Reply:
I would say no as they originally weren't part of it, and I think float16 can be a bit tricky to deal with sometimes. We already skip a bunch of float16 test instances.

Reply:
We could just put them in there and see how many failures we get.

Reply:
It's likely that the universe will break. We could take the approach of adding it "strategically" when needed. Up to you.